# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/netpbm/hotfix-shared.patch
# Copyright (C) 2004 - 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- netpbm-10.25/buildtools/installnetpbm.pl	2003-11-05 23:35:37.000000000 +0100
+++ netpbm-10.25-t2/buildtools/installnetpbm.pl	2004-10-18 17:46:15.739386208 +0200
@@ -146,13 +146,12 @@
 
     if (`cp --version 2>/dev/null` =~ m/GNU/) {
         # It's GNU Cp -- we have options galore, and they're readable.
-        $cpCommand = "cp --recursive --preserve --no-dereference";
+        $cpCommand = "cp --preserve --no-dereference";
     } else {
         # This works on Cp from "4th Berkeley Distribution", July 1994.
         # Mac OSX has this.
-        # -R means recursive with no dereferencing of symlinks
         # -p means preserve attributes
-        $cpCommand = "cp -R -p";
+        $cpCommand = "cp -p -d";
     }
     return($cpCommand);
 }
